tests/sqlparser_postgres.rs

```diff
@@ -300,6 +300,33 @@ fn parse_bad_if_not_exists() {
     );
 }
 
+#[test]
+fn parse_create_schema_if_not_exists() {
+    let sql = "CREATE SCHEMA IF NOT EXISTS schema_name";
+    let ast = pg().verified_stmt(sql);
```

Comment:
Please default to `pg_and_generic()` unless testing a construct that can't be parsed by the generic parser. The idea of the generic parser is to be able to parse as much SQL code as possible.